Dockerfile

```diff
@@ -7,7 +7,10 @@ FROM node:14-alpine3.13 as base
 LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
 
 ENV TZ=Europe/London
-RUN ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime && echo "$TZ" > /etc/timezone
+RUN apk add --no-cache tzdata && \
+      test -e "/usr/share/zoneinfo/$TZ" && \
```

Comment:
Instead of this line, we could also add a "docker build test" that would test the timezone by running the `date` command:
```
$ docker run --rm -ti <built_tag> date | egrep "(GMT|BST)"
Thu Apr  8 15:55:29 BST 2021
# exit status 0
```
vs
```
$ k exec -ti hmpps-interventions-ui-5fc67cffcd-z4kgx -- date | egrep "(GMT|BST)"
# exit status 1, thus failing the build
```
However, this isn't trivial to chain after the docker build, before the push at the moment